_use_cases/schema_validation_and_enforecement.markdown

```diff
@@ -68,7 +68,7 @@ even if producing applications are misconfigured.
 
 The filter currently supports two modes of operation:
 
-1. Schema validation[^3] validates the content of the record against a schema. Use this for topics which have an entry in
+1. Schema validation validates the content of the record against a schema. Use this for topics which have an entry in
```

Comment:
The feature is limited to JSON schemas, current, so I think the footnote remains correct.